### ISSUE TITLE
Add missing labels on all interactable elements

### ISSUE THREAD

**Issue by ChiuchiuSorin** [xwikisas/application-admintools#105]
Some links and buttons are missing labels needed for screen readers to better explain the functionality of an element.
